app/components/post_list/post/body/content/message_attachments/action_menu/index.tsx

```diff
@@ -25,21 +25,22 @@ const ActionMenu = ({dataSource, defaultOption, disabled, id, name, options, pos
     }
     const [selected, setSelected] = useState(isSelected?.value);
 
-    const handleSelect = useCallback(async (selectedItem: string | string[]) => {
-        if (Array.isArray(selectedItem)) { // Since AutocompleteSelector is not multiselect, we are sure we only receive a string
+    const handleSelect = useCallback(async (selectedItem: DialogOption | DialogOption[] | undefined) => {
```

Comment:
`DialogOption | DialogOption[] | undefined` there was a type defined somewhere else for this.

Perhaps we should define all those types in the types folder.. it can be done in a follow up PR